package.json

```diff
@@ -5,17 +5,14 @@
   "description": "The BRP application",
   "scripts": {
     "start": "node .",
-    "start:dev":
-      "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
+    "start:dev": "./node_modules/parallelshell/index.js 'DEBUG=* ./node_modules/nodemon/bin/nodemon.js .' 'npm run watch:scss'",
     "watch:scss": "./node_modules/nodemon/bin/nodemon.js -e scss -x 'npm run sass'",
     "test": "NODE_ENV=test mocha",
     "lint": "./node_modules/eslint/bin/eslint.js .",
     "style": "./node_modules/jscs/bin/jscs **/*.js --config=./.jscsrc.json",
-    "quality":
-      "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
+    "quality": "./node_modules/plato/bin/plato -r -x 'node_modules|reports|test' -d reports/plato .",
     "postinstall": "npm run sass",
-    "sass":
-      "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./dist/css/app.css --include-path ./node_modules"
+    "sass": "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./dist/css/app.css --include-path ./node_modules"
```

Comment:
As above. 

Also, you may want to be "a ittle bit" careful with `--include-path ./node_modules` as well, as it makes assumptions about npm install locations which are not always valid (although this mostly applies to packages which are themselves installed as child modules). In particular, if a module shares a dependency with an ancestor then it won't be in `./node_modules`, but potentially in `../../node_modules`. See https://github.com/sass/node-sass/issues/949

Here's a thing I'm working on to get around this - https://github.com/lennym/npm-sass